modules/dataquery/jsx/react.app.js

```diff
@@ -180,18 +180,22 @@ class DataQueryApp extends Component {
     for (let key in this.state.queryIDs) {
       for (let i = 0; i < this.state.queryIDs[key].length; i += 1) {
         let curRequest;
+        const document = encodeURIComponent(this.state.queryIDs[key][i]);
         curRequest = Promise.resolve(
-          $.ajax(loris.BaseURL + '/AjaxHelper.php?Module=dataquery&script=GetDoc.php&DocID=' + encodeURIComponent(this.state.queryIDs[key][i])), {
-            data: {
-              DocID: this.state.queryIDs[key][i]
-            },
-            dataType: 'json'
-          }).then((value) => {
-          let queries = this.state.savedQueries;
-
-          queries[value._id] = value;
-          this.setState({savedQueries: queries});
-        });
+          fetch(
+            `${window.location.origin}/dataquery/documents/${document}`,
```

Comment:
is there a reason to be using this instead of `loris.BaseURL`?

Reply:
@HenriRabalais the use of the dollar sign here isn't jQuery. See here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
he's not asking about the dollar sign, he's asking why it isn't using "`loris.baseURL +  '/dataquery/etc`"

Reply:
How would window.location.origin work if the instance isn't hosted on the root of the domain?

Reply:
@driusan @HenriRabalais  `console.log(window.location.origin);` when visiting a site with a subdomain such as docs.google.com returns `https://docs.google.com` on chrome, safari, and firefox.